test/navigation/NavigatorSpec.scala

```diff
@@ -57,16 +58,15 @@ class NavigatorSpec extends BaseSpec with ScalaCheckPropertyChecks {
 
     "check returns page" - {
 
-      "navigate to how many memebers" - {
+      "navigate to check return date" - {
```

Comment:
Test name wrong, this navigates to the active bank account page not the check return date page